src/threadpool-pthreads.c

```diff
@@ -424,6 +429,11 @@ static struct pthreadpool* pthreadpool_allocate(size_t threads_count) {
 		if (threadpool == NULL) {
 			return NULL;
 		}
+	#elif defined(_WIN32)
+		threadpool = _aligned_malloc(threadpool_size, PTHREADPOOL_CACHELINE_SIZE);
```

Comment:
Per MSDN, `_aligned_malloc` requires `malloc.h` include.

Reply:
Sure. Will fix.

Reply:
BTW, Should I better to send new PR to change commit-id for XNNPACK in later?

https://github.com/google/XNNPACK/blob/master/cmake/DownloadPThreadPool.cmake#L15

Reply:
Several files need to be updated:
- https://github.com/google/XNNPACK/blob/master/cmake/DownloadPThreadPool.cmake
- https://github.com/google/XNNPACK/blob/master/WORKSPACE#L47-L51
- https://github.com/tensorflow/tensorflow/blob/master/tensorflow/workspace.bzl#L149-L157
- https://github.com/tensorflow/tensorflow/blob/master/third_party/pthreadpool/workspace.bzl

Reply:
Okay, will do after this will be merged.